15-transformer.ipynb

```diff
@@ -354,10 +354,10 @@
     "        [p / np.power(10000, 2 * (j // 2) / dim) for j in range(dim)]\n",
     "        for p in range(nb_p)\n",
     "    ])\n",
-    "    E[:, 0::2] = torch.FloatTensor(np.sin(theta[:, 0::2]))\n",
-    "    E[:, 1::2] = torch.FloatTensor(np.cos(theta[:, 1::2]))\n",
     "    E.detach_()\n",
```

Comment:
This line is unnecessary, right?
In particular, we are not learning the embeddings.